target/linux/ramips/dts/RT-AC51U.dts

```diff
@@ -116,6 +116,20 @@
 	ralink,mtd-eeprom = <&factory 0>;
 };
 
+&pcie {
```

Comment:
Please drop this change from your PR! It is unrelated to "add support for Asus RT-AC54U".

@Redfoxmoon3 Is this change required at all for the AC51U?